protocols/umiToFastq.sh

```diff
@@ -0,0 +1,70 @@
+#MOLGENIS nodes=1 ppn=4 mem=4gb walltime=05:59:00
+
+#Parameter mapping
+#string seqType
+#string peEnd1BarcodeFqGz
+#string peEnd2BarcodeFqGz
+#string srBarcodeFqGz
+#string intermediateDir
+#string TrimGaloreVersion
+#string project
+#string groupname
+#string tmpName
+#string externalSampleID
+#string lane
+#string logsDir
+#string projectRawtmpDataDir
+#string projectQcDir
+
+#Echo parameter values
+echo "seqType: ${seqType}"
+echo "peEnd1BarcodeFqGz: ${peEnd1BarcodeFqGz}"
+echo "peEnd2BarcodeFqGz: ${peEnd2BarcodeFqGz}"
+echo "srBarcodeFqGz: ${srBarcodeFqGz}"
+echo "intermediateDir: ${intermediateDir}"
+echo "outputDir: ${projectRawtmpDataDir}"
+
+module load "${TrimGaloreVersion}"
+module list
+
+
+#
+## UMI tools https://www.biostars.org/p/357359/
+#
+https://umi-tools.readthedocs.io/en/latest/QUICK_START.html#step-1-install-umi-tools
+
+#If paired-end do cutadapt for both ends, else only for one, and fastQC calculations.
+if [ ${seqType} == "PE" ]
```

Comment:
quotes

Reply:
protocol wordt niet gebruikt, is een reminder om deze een keer in te bouwen.